git/git.go

```diff
@@ -249,6 +249,16 @@ func (git *Git) RemoteAdd(name string, url string) error {
 	return err
 }
 
+// HasRemotes returns if a there are any remotes configured.
+func (git *Git) HasRemotes() (bool, error) {
+	res, err := git.exec("config", "--get-regexp", "remote\\.")
```

Comment:
Did you consider using `git remote` instead of `git config --get-regexp remote \\.`?

Reply:
Yes, I was googling a bit for a Git plumbing command that does this as I figured `git remote` is porcelain. I ended up using `git config`, not sure if this is any better - open to suggestions what the most stable way to detect this would be.

Reply:
Ok got it, I guess the git config command is less likely to change over time so we can keep it as it is.